public/_includes/_util-fns.jade

```diff
@@ -77,35 +78,21 @@ mixin makeExample(_filePath, region, _title, stylePatterns)
   code-example(language="#{language}" format="#{format}")
     != styleString(frag, stylePatterns)
 
-//- Like makeExample, but the first argument is a path that is
-//- relative to the project root. Unless title is defined,
-//- the project relative path will be used.
-mixin makeProjExample(projRootRelativePath, region, title, stylePatterns)
-  - var relPath = projRootRelativePath.trim();
-  - var filePath = getExampleName() + '/ts/' + relPath;
-  - if (!title) {
-  -   // Is path like styles.1.css? Then drop the '.1' qualifier:
-  -   var matches = relPath.match(/^(.*)\.\d(\.\w+)$/);
-  -   title = matches ? matches[1] + matches[2] : relPath;
-  - }
-  +makeExample(filePath, region, title, stylePatterns)
-
-//- Like makeExample, but doesn't show line numbers, and the first
-//- argument is a path that is relative to the example project root.
-//- Unless title is defined, the project relative path will be used.
-//- Title will always end with a phrase in parentheses; if no such
-//- ending is given, then the title will be suffixed with
-//- either "(excerpt)", or "(#{region})" when region is defined.
-mixin makeExcerpt(projRootRelativePath, region, title, stylePatterns)
-  - var relPath = projRootRelativePath.trim();
-  - var filePath = getExampleName() + '/ts/' + relPath;
-  - if (!title) {
-  -   // Is path like styles.1.css? Then drop the '.1' qualifier:
-  -   var matches = relPath.match(/^(.*)\.\d(\.\w+)$/);
-  -   title = matches ? matches[1] + matches[2] : relPath;
-  - }
-  - var excerpt = region || 'excerpt';
-  - if (title && !title.match(/\([\w ]+\)$/)) title = title + ' (' + excerpt + ')';
+//- Like makeExample, but: (1) doesn't show line numbers. (2) If region
+//- is omitted and title is 'foo (r)' then region is taken as 'r'.
+//- (3) Title will always end with a phrase in parentheses; if no such
+//- ending is given or is just (), then the title will be suffixed with
+//- either "(excerpt)", or "(#{_region})" when _region is defined.
+mixin makeExcerpt(_filePath, _region, _title, stylePatterns)
+  - var matches = _filePath.match(/(.*)\s+\(([\w ]*)\)$/);
+  - var parenText;
+  - if (matches) { _filePath = matches[1]; parenText = matches[2]; }
+  - var adjustments = adjustExamplePathAndTitle({filePath:_filePath, title:_title});
```

Comment:
I can't move lines 90-92 to the top because line 89 potentially modifies `_filePath`.